src/main/java/org/gridsuite/securityanalysis/server/entities/AbstractLimitViolationEntity.java

```diff
@@ -0,0 +1,54 @@
+package org.gridsuite.securityanalysis.server.entities;
+
+import com.powsybl.iidm.network.Branch;
+import com.powsybl.security.LimitViolationType;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@NoArgsConstructor
+@Getter
+@MappedSuperclass
+public abstract class AbstractLimitViolationEntity {
+
+    @Id
+    @GeneratedValue
+    private UUID id;
+
+    @ManyToOne
+    private ConstraintEntity constraint;
+
+    private String subjectName;
```

Comment:
Why here ?
Move subjectName in the  ConstraintEntity ?

Reply:
it is currently technically possible to have a different subjectName for the same subjectId
it would remove this possibility